src/MenuItem.jsx

```diff
@@ -161,10 +161,13 @@ export class MenuItem extends React.Component {
         role: 'option',
         'aria-selected': props.isSelected,
       };
-    } else if (props.role === null) {
+    } else if (props.role === null || props.role === 'none') {
```

Comment:
I left the `null` option in here for backward compatibility, but I'm also ok with removing it.